opal/opal/base.rb

```diff
@@ -4,6 +4,7 @@
 require 'corelib/class'
 require 'corelib/basic_object'
 require 'corelib/kernel'
+require 'corelib/kernel/format'
```

Comment:
The require should go into `opal.rb`, `opal/base.rb` is meant to require the bare minimum for opal to function.

Reply:
But then it's a breaking change, since `Kernel#format` and `Kernel#sprintf` were part of `corelib/kernel` (which is required in `opal/base.rb`).

Reply:
Seems like an acceptable breaking change to me.

Reply:
Ok done :+1: